src/components/FilterList.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { firestore } from '../lib/firebase';
+import { useCollection } from 'react-firebase-hooks/firestore';
+
+let arrString = ['fruit', 'vegetables', 'water', 'legumes'];
```

Comment:
Any reason this is using `let`?

```suggestion
const arrString = ['fruit', 'vegetables', 'water', 'legumes'];
```